NetKet/Sampler/py_sampler.hpp

```diff
@@ -86,14 +88,20 @@ void AddSamplerModule(py::module &m) {
                     &AbstractSampler::SetVisible,
                     R"EOF(
                       numpy.array: The quantum numbers being sampled,
-                       and distributed according to $$|\Psi(v)|^2$$ )EOF")
+                       and distributed according to $$F(\Psi(v))$$ )EOF")
       .def_property_readonly("acceptance", &AbstractSampler::Acceptance, R"EOF(
         numpy.array: The measured acceptance rate for the sampling.
         In the case of rejection-free sampling this is always equal to 1.  )EOF")
       .def_property_readonly("hilbert", &AbstractSampler::GetHilbert, R"EOF(
         netket.hilbert: The Hilbert space used for the sampling.  )EOF")
       .def_property_readonly("machine", &AbstractSampler::GetMachine, R"EOF(
-        netket.machine: The machine used for the sampling.  )EOF");
+        netket.machine: The machine used for the sampling.  )EOF")
+      .def_property("machine_func", &AbstractSampler::GetMachineFunc,
```

Comment:
Is this property still causing run-time errors? (I can't check it myself right now, unfortunately). If so, probably better to remove it for now.

Reply:
no, it works... there is also a unit test that checks that, we can leave it